src/styles/input-group.scss

```diff
@@ -78,19 +85,29 @@ $fd-input-border-radius: var(--sapField_BorderCornerRadius);
     }
   }
 
+  .fd-tokenizer {
```

Comment:
This should be done in tokenizer.scss

Reply:
this is to remove the background for when the tokenizer is wrapped inside the input-group only

Reply:
This breaks our rule that you can not set properties of one component from another component's scss file. And I see that you have set the background to none in tokenizer.scss so you should be fine to delete this bit